packages/camera/camera_avfoundation/ios/camera_avfoundation.podspec

```diff
@@ -13,9 +13,13 @@ A Flutter plugin to use the camera from your Flutter app.
   s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :http => 'https://github.com/flutter/packages/tree/main/packages/camera_avfoundation' }
   s.documentation_url = 'https://pub.dev/packages/camera_avfoundation'
-  s.source_files = 'camera_avfoundation/Sources/camera_avfoundation/**/*.{h,m}'
-  s.public_header_files = 'camera_avfoundation/Sources/camera_avfoundation/include/**/*.h'
-  s.module_map = 'camera_avfoundation/Sources/camera_avfoundation/include/CameraPlugin.modulemap'
```

Comment:
it's been a while since i touch this. Why we are removing the module map?

Reply:
It's not supported with Swift static libraries.
```
[!] Using Swift static libraries with custom module maps is currently not supported. Please build `camera_avfoundation` as a framework or remove the custom module map.
```

It's still the same issue you've encountered here https://github.com/flutter/plugins/pull/6369

Reply:
Oh i completely forgot about this! thanks for the reference.